adserver/templates/adserver/reports/includes/publisher-report-table.html

```diff
@@ -4,19 +4,18 @@
 <table class="table table-hover report">
   <thead>
     <tr>
-      <th><strong>{% trans 'Report Index' %}</strong></th>
-      <th class="text-right"><strong>{% trans 'Total&nbsp;Views' %}</strong></th>
-      <th class="text-right"><strong>{% trans 'Total&nbsp;Clicks' %}</strong></th>
-      <th class="text-right"><strong>{% blocktrans %}Total&nbsp;<abbr title="Click through rate">CTR</abbr>{% endblocktrans %}</strong></th>
+      <th><strong>{% trans 'Index' %}</strong></th>
+      <th class="text-right"><strong>{% trans 'Views' %}</strong></th>
+      <th class="text-right"><strong>{% trans 'Clicks' %}</strong></th>
+      <th class="text-right"><strong>{% blocktrans %}<abbr title="Click through rate">CTR</abbr>{% endblocktrans %}</strong></th>
+      <th class="text-right"><strong>{% trans 'Revenue' %}</strong></th>
       {% if request.user.is_staff %}
```

Comment:
Overall, I like that all the staff specific things are grouped together. That's really nice. However, I don't like the divider very much. I put some thoughts in #328.

Reply:
Yea -- just my naive implementation. Thanks for improving :)